tests/x11/vnc_two_passwords.pm

```diff
@@ -182,7 +189,9 @@ sub run {
         type_string("$display");
         send_key("ret");
         # We first test for a unsucessfull login
-        assert_screen('tigervnc-desktop-login');
+        # increase timeout on aarch64 for assert_screen, see poo#119416
+        my $timeout = (is_aarch64 && is_sle) ? 90 : 30;
```

Comment:
I can see several functions use the same variable $timeout， can you please set it as global variable?

Reply:
yes, good idea. forgot to set this since there are some places need to be covered, thanks!